app/jobs/data_exports/export_resources_job.rb

```diff
@@ -0,0 +1,9 @@
+module DataExports
+  class ExportResourcesJob < ApplicationJob
+    queue_as :data_export
```

Comment:
Does this queue already exists?

Reply:
no, this is a new queue.

I would not mix this kind of jobs with other jobs as the time building the files could be long enough to add delays to the queue processing.